benchmarks/harness/controller.py

```diff
@@ -210,6 +213,10 @@ def verifyStats(self):
                         help='Disable LSAN')
     parser.add_argument('--build', type=str, default="",
                         help='Path to the build dir')
+    parser.add_argument('--seed', type=int, default=123,
+                        help='Random seed (default: enabled)')
+    parser.add_argument('--splat-to-random', type=int, default=1,
```

Comment:
`tpp-run` has some options (`random`, `normal`, etc) and `normal` is the preferred here. Maybe this should be the same list, with default `normal`.